src/ensembl/src/header/Header.scss

```diff
@@ -6,39 +6,72 @@
   justify-content: space-between;
   align-items: center;
   color: $white;
-  padding: 2px 20px;
+  height: 28px;
+  padding: 0 30px 0 22px;
   background-color: $black;
+
+  &Left {
+    display: flex;
+    align-items: center;
+    line-height: 1;
+  }
+
+  &LeftTextBlock {
+    display: flex;
+    align-items: center;
+    margin-top: 1px;
```

Comment:
This definition here only seems to affect the logotype so can we can remove this and increase the `margin-top` value on line 38 to `3px`?

Reply:
Yeah, the way the combined effect of the several flexboxes the topbar is very strange now. I've moved the margin-top to the `logotypeWrapper` class.